src/spec/PointerInputSpec.ts

```diff
@@ -0,0 +1,111 @@
+/// <reference path="jasmine.d.ts" />
+
+/// <reference path="TestUtils.ts" />
+
+function executeMouseEvent(type: string, button: ex.Input.PointerButton, x: number, y: number, target: HTMLElement) {
```

Comment:
Sorry, one last change. If you move this function inside the `describe` function, it can access `engine.canvas`, eliminating the extra argument. Additionally, TS supports default params so you can set `x = 0` and `y = 0` to eliminate other duplication below too.